lib/linter.js

```diff
@@ -252,7 +220,7 @@ class Linter {
 
     let currentSource = options.source;
 
-    let pending = this.statusForModule('pending', options.moduleId);
+    let pending = getConfigForFile(this.config, options).pending;
```

Comment:
We use `getConfigForFile` just below, we should capture it in a variable instead of calling it twice.

```suggestion
    let config = getConfigForFile(this.config, options);
    let pending = config.pending;
```

Reply:
:|... oops!